lib/src/components/NxThreatCounter/NxThreatCounter.tsx

```diff
@@ -13,30 +13,47 @@ export { Props } from './types';
 import './NxThreatCounter.scss';
 
 export default function NxThreatCounter({
-  criticalCount, severeCount, moderateCount, lowCount, layout, className, ...attrs }: Props) {
+  criticalCount, severeCount, moderateCount, lowCount, noneCount, layout, className, ...attrs }: Props) {
 
   const layoutClasses = classnames('nx-threat-counter-container', className, {
     [`nx-threat-counter-container--${layout}`]: layout
   });
 
+  if (typeof criticalCount !== 'number' &&
+      typeof severeCount !== 'number' &&
+      typeof moderateCount !== 'number' &&
+      typeof lowCount !== 'number' &&
+      typeof noneCount !== 'number') {
+    return null;
```

Comment:
I'd log a console warning here as it's not really a valid use of the component

Reply:
> I'd log a console warning here as it's not really a valid use of the component

Added in https://github.com/sonatype/sonatype-react-shared-components/pull/418/commits/ec3631afe405fb01d0c9406fc4d0e68496ddb5da